ci/scale-run.sh

```diff
@@ -64,25 +64,33 @@ $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
 $OVNSUDO docker exec ovn-rally rally-ovs task start /root/rally-ovn/workload/create_networks.json
 TASKID=$($OVNSUDO docker exec ovn-rally rally task list --uuids-only)
 $OVNSUDO docker exec ovn-rally rally task report $TASKID --out /root/create-networks-output.html
+$OVNSUDO bash -c "docker exec ovn-rally rally task results $TASKID > /root/create-networks-data.json"
```

Comment:
The redirection is the issue, try adding "--out /root/create-networks-data.json"